recipe/migrations/window_cuda.yaml

```diff
@@ -0,0 +1,41 @@
+migrator_ts: 1604255168
+__migrator:
+  kind:
+    version
+  migration_number:
+    1
+  build_number:
+    1
+
+c_compiler:
+  - gcc                        # [linux]
+  - clang                      # [osx]
+  # part of a zip_keys: c_compiler, cxx_compiler, cuda_compiler_version
+  - vs2015                     # [win]
```

Comment:
We don't support vs2015 anymore

Reply:
That will simplify the migration, but unfortunately it'll mean that CUDA 9.2 won't get support on Windows :/

Reply:
True, but @conda-forge/core made a decision to stop supporting it and going back is not possible without rebuilding all of the dependencies of a package needing CUDA 9.2.

Reply:
The CUDA 9.2 headers [could be patched](https://github.com/LeelaChessZero/lczero.org/issues/12#issue-572284272), but that doesn't look like a good idea.

Reply:
Technically you can make it work by installing vc140 toolchain inside VS2017 and making sure `vs2015_win-64` use the VS2017 installation of vc140.